playbooks/eucalyptus_basic_server.yml

```diff
@@ -3,7 +3,7 @@
 # This file based on `edx_sandbox.yml`
 
 - name: Configure instance(s)
-  hosts: all
+  hosts: edxapp-server
```

Comment:
Does this mean that we'll have to run both the basic server and then the insights playbook to get them both setup for a basic server installation?